lib/tmdb/client.rb

```diff
@@ -21,11 +21,15 @@ def get_movie_search_results(movie_title)
       end
 
       def get_movies_for_actor(actor_name:, page:, sort_by:)
+        page = page.presence || 1
+        sort_by = sort_by.presence || 'popularity'
         person_data = request(:person_search, query: actor_name)[:results]&.first
 
-        return OpenStruct.new(
-          not_found_message: "No actors found for '#{actor_name}'."
-        ) if person_data.blank?
+        if person_data.blank?
+          return OpenStruct.new(
+            not_found_message: "No actors found for '#{actor_name}'."
+          )
+        end
```

Comment:
I know you prefer a `return` block with a trailing `if`, but rubocop converted it to this style. If you prefer, I can change it back.

Reply:
I actually don't have a strong preference, so 👍